src/UKHO.S100PermitService.API/Controllers/PermitController.cs

```diff
@@ -29,44 +31,39 @@ public PermitController(IHttpContextAccessor httpContextAccessor, ILogger<Permit
         /// Provide Permits for requested licence Id.
```

Comment:
Please update this summary as we are not using any reference of licence now.

Reply:
Resolved